### PR TITLE
check lesson 1

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson1/program1.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const myfunction= (a,b)=>{
+  c= a+b;
+  return c;
+}
+module.exports=myfunction;
+console.log(myfunction(5,10));
```
Comment (songz) [inline lesson1/program1.js]: remove console.log

File: lesson1/program1.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const myfunction= (a,b)=>{
+  c= a+b;
```
Comment (songz) [inline lesson1/program1.js]: `const c = ` always declare your variables with either a `const` or `let`
also, pick better variable names. `c` could be better renamed as `sum`

File: lesson1/program1.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const myfunction= (a,b)=>{
```
Comment (songz) [inline lesson1/program1.js]: myfunction tells us nothing about what the function does. you should use better names  like... `add`?

File: lesson1/testprogram1.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const G= require('./program1.js');
+
+const test=(a,b,c)=>{
+  if(G(a,b)===c){
+  console.log('You are CORRECT');
+  }else {console.log('You are INCORRECT');
```
Comment (songz) [inline lesson1/testprogram1.js]: move console.log to new line. indentation looks pretty bad

Comment (kinuesantos) [reply lesson1/testprogram1.js]: Thanks for the feedback. I updated both files.

File: lesson1/program1.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const adding= (a,b)=>{
+  total= a+b;
```
Comment (songz) [inline lesson1/program1.js]: please declare your variables. `const total = a+b;`

File: lesson1/testprogram1.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+const G= require('./program1.js');
```
Comment (songz) [inline lesson1/testprogram1.js]: could you give G a better name? maybe... `sum` or `add`

File: lesson1/testprogram1.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+const test=(a,b,c)=>{
```
Comment (songz) [inline lesson1/testprogram1.js]: could you give a,b,c better name? ie: input1,input2, correctAnswer

**Review by songz — APPROVED** [review]

**Review by saumya665 — APPROVED** [review]

**Pull request merged**